src/utils.js

```diff
@@ -127,3 +127,12 @@ export function partition(array, n) {
 export function isUUID4(uuid) {
   return RE_UUID.test(uuid);
 }
+
+/**
+ * Returns the current UNIX timestamp.
+ *
+ * @return {Number}
+ */
+export function getUnixTime() {
+  return Math.floor(new Date().getTime() / 1000);
```

Comment:
Maybe we should have everything in ms on the client side, since ``Date.now()`` return milliseconds, we can just put backoffRelease in ms and not bother about ms to s conversion.